fragments/header/template.vue

```diff
@@ -47,6 +47,9 @@
         },
         mounted: function() {
           this.menuHeight = this.$refs.autoHeight.clientHeight;
+          window.addEventListener('pageRendered', (e) => {
+              this.toggleMenu();
```

Comment:
@smcgrath0 this seems wrong - this would mean to toggle the menu every time a page change happens. so if yuo navigate from one page to another and the menu is closed it would open.